### PR TITLE
Downgrading to NET46

### LINKED ISSUE #141: Possibly reduce target from net462 to net46
Just a small follow up on your targeting. I noticed you needed to bump up the version to 4.6.x due to `RSA.SignData`, however you targeted 4.6.2 which forced my 4.6.1 project into `netstandard` which I didn't want.

Is there any reason why you need 4.6.2 specifically? If not you should target the minimum required version (being `net46`).

### PR DESCRIPTION
**Pull request by abatishchev**
Resolves #141.

### CODE REVIEW

**Comment by abatishchev** [comment]
@psylenced can you please try this package? https://ci.appveyor.com/project/abatishchev/jwt/build/1.0.187/artifacts

File: src/JWT/JWT.csproj [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>net462;netstandard1.3</TargetFrameworks>
```
Comment (psylenced) [inline src/JWT/JWT.csproj]: @abatishchev I believe you missed this line. The output still targets 4.6.2.

Comment (abatishchev) [reply src/JWT/JWT.csproj]: Oh, right, redid. Please try a package from the most recent build (once it's done)

**Comment by psylenced** [comment]
I haven't installed the file, but based on looking inside the `JWT.3.1.1-beta.nupkg` file - the `JWT.nuspec` and `lib` folders are both referencing the 4.6 version now.

So it looks good to me.

**Comment by abatishchev** [comment]
Can you please download the package, add a new local nuget feed in VS, try installing (update with pre-releases) it in your project? Just to make sure everything is fine.

**Comment by psylenced** [comment]
Looks good do me.

(Apologies for the line spam)


v188:

> Attempting to gather dependency information for package 'JWT.3.1.1-beta' with respect to project 'Application.Utils', targeting '.NETFramework,Version=v4.6.1'
Gathering dependency information took 238.82 ms
Attempting to resolve dependencies for package 'JWT.3.1.1-beta' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'JWT.3.1.1-beta'
Resolved actions to install package 'JWT.3.1.1-beta'
Retrieving package 'JWT 3.1.1-beta' from 'E:\Downloads\feed'.
Removed package 'JWT.3.0.1' from 'packages.config'
Successfully uninstalled 'JWT.3.0.1' from Application.Utils
Adding package 'JWT.3.1.1-beta' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'JWT.3.1.1-beta' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'JWT.3.1.1-beta' to 'packages.config'
Successfully installed 'JWT 3.1.1-beta' to Application.Utils
Removing package 'JWT.3.0.1' from folder 'C:\Application\Source\Application.APP\packages'
Removed package 'JWT.3.0.1' from folder 'C:\Application\Source\Application.APP\packages'
Executing nuget actions took 1.01 sec
Time Elapsed: 00:00:01.3067605


v187:

<sub>Attempting to gather dependency information for package 'JWT.3.1.1-beta' with respect to project 'Application.Utils', targeting '.NETFramework,Version=v4.6.1'
Gathering dependency information took 23.2 sec
Attempting to resolve dependencies for package 'JWT.3.1.1-beta' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'JWT.3.1.1-beta'
Resolved actions to install package 'JWT.3.1.1-beta'
Retrieving package 'JWT 3.1.1-beta' from 'E:\Downloads\feed'.
Retrieving package 'Microsoft.NETCore.Platforms 1.1.0' from 'nuget.org'.
Retrieving package 'Microsoft.Win32.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'NETStandard.Library 1.6.1' from 'nuget.org'.
Retrieving package 'System.AppContext 4.3.0' from 'nuget.org'.
Retrieving package 'System.Collections 4.3.0' from 'nuget.org'.
Retrieving package 'System.Collections.Concurrent 4.3.0' from 'nuget.org'.
Retrieving package 'System.Console 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Debug 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.DiagnosticSource 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Tools 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Tracing 4.3.0' from 'nuget.org'.
Retrieving package 'System.Globalization 4.3.0' from 'nuget.org'.
Retrieving package 'System.Globalization.Calendars 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.Compression 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.Compression.ZipFile 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.FileSystem 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.FileSystem.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq.Expressions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Http 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Sockets 4.3.0' from 'nuget.org'.
Retrieving package 'System.ObjectModel 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reflection 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reflection.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reflection.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Resources.ResourceManager 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Handles 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Numerics 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Encoding 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.X509Certificates 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.Encoding 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.Encoding.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.RegularExpressions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Tasks 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Timer 4.3.0' from 'nuget.org'.
Retrieving package 'System.Xml.ReaderWriter 4.3.0' from 'nuget.org'.
Retrieving package 'System.Xml.XDocument 4.3.0' from 'nuget.org'.
Removed package 'JWT.3.0.1' from 'packages.config'
Successfully uninstalled 'JWT.3.0.1' from Application.Utils
Adding package 'Microsoft.NETCore.Platforms.1.1.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'Microsoft.NETCore.Platforms.1.1.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'Microsoft.NETCore.Platforms.1.1.0' to 'packages.config'
Successfully installed 'Microsoft.NETCore.Platforms 1.1.0' to Application.Utils
Adding package 'Microsoft.Win32.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'Microsoft.Win32.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'Microsoft.Win32.Primitives.4.3.0' to 'packages.config'
Successfully installed 'Microsoft.Win32.Primitives 4.3.0' to Application.Utils
Adding package 'System.AppContext.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.AppContext.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.AppContext.4.3.0' to 'packages.config'
Successfully installed 'System.AppContext 4.3.0' to Application.Utils
Adding package 'System.Collections.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Collections.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Collections.4.3.0' to 'packages.config'
Successfully installed 'System.Collections 4.3.0' to Application.Utils
Adding package 'System.Collections.Concurrent.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Collections.Concurrent.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Collections.Concurrent.4.3.0' to 'packages.config'
Successfully installed 'System.Collections.Concurrent 4.3.0' to Application.Utils
Adding package 'System.Console.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Console.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Console.4.3.0' to 'packages.config'
Successfully installed 'System.Console 4.3.0' to Application.Utils
Adding package 'System.Diagnostics.Debug.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Debug.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Debug.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Debug 4.3.0' to Application.Utils
Adding package 'System.Diagnostics.DiagnosticSource.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.DiagnosticSource.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.DiagnosticSource.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.DiagnosticSource 4.3.0' to Application.Utils
Adding package 'System.Diagnostics.Tools.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Tools.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Tools.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tools 4.3.0' to Application.Utils
Adding package 'System.Diagnostics.Tracing.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Tracing.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Diagnostics.Tracing.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tracing 4.3.0' to Application.Utils
Adding package 'System.Globalization.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Globalization.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Globalization.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization 4.3.0' to Application.Utils
Adding package 'System.Globalization.Calendars.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Globalization.Calendars.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Globalization.Calendars.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization.Calendars 4.3.0' to Application.Utils
Adding package 'System.IO.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.4.3.0' to 'packages.config'
Successfully installed 'System.IO 4.3.0' to Application.Utils
Adding package 'System.IO.Compression.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.Compression.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.Compression.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression 4.3.0' to Application.Utils
Adding package 'System.IO.Compression.ZipFile.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.Compression.ZipFile.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.Compression.ZipFile.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression.ZipFile 4.3.0' to Application.Utils
Adding package 'System.IO.FileSystem.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.FileSystem.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.FileSystem.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem.Primitives 4.3.0' to Application.Utils
Adding package 'System.IO.FileSystem.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.FileSystem.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.IO.FileSystem.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem 4.3.0' to Application.Utils
Adding package 'System.Linq.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Linq.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Linq.4.3.0' to 'packages.config'
Successfully installed 'System.Linq 4.3.0' to Application.Utils
Adding package 'System.Linq.Expressions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Linq.Expressions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Linq.Expressions.4.3.0' to 'packages.config'
Successfully installed 'System.Linq.Expressions 4.3.0' to Application.Utils
Adding package 'System.Net.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Primitives 4.3.0' to Application.Utils
Adding package 'System.Net.Sockets.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Sockets.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Sockets.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Sockets 4.3.0' to Application.Utils
Adding package 'System.ObjectModel.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.ObjectModel.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.ObjectModel.4.3.0' to 'packages.config'
Successfully installed 'System.ObjectModel 4.3.0' to Application.Utils
Adding package 'System.Reflection.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection 4.3.0' to Application.Utils
Adding package 'System.Reflection.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Extensions 4.3.0' to Application.Utils
Adding package 'System.Reflection.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.Primitives.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Reflection.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Primitives 4.3.0' to Application.Utils
Adding package 'System.Resources.ResourceManager.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Resources.ResourceManager.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Resources.ResourceManager.4.3.0' to 'packages.config'
Successfully installed 'System.Resources.ResourceManager 4.3.0' to Application.Utils
Adding package 'System.Runtime.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime 4.3.0' to Application.Utils
Adding package 'System.Runtime.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Extensions 4.3.0' to Application.Utils
Adding package 'System.Runtime.Handles.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Handles.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Handles.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Handles 4.3.0' to Application.Utils
Adding package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices 4.3.0' to Application.Utils
Adding package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' to Application.Utils
Adding package 'System.Runtime.Numerics.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Numerics.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Runtime.Numerics.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Numerics 4.3.0' to Application.Utils
Adding package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Encoding 4.3.0' to Application.Utils
Adding package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.X509Certificates 4.3.0' to Application.Utils
Adding package 'System.Net.Http.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Http.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Net.Http.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Http 4.3.0' to Application.Utils
Adding package 'System.Text.Encoding.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.Encoding.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding 4.3.0' to Application.Utils
Adding package 'System.Text.Encoding.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.Encoding.Extensions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.Encoding.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding.Extensions 4.3.0' to Application.Utils
Adding package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.RegularExpressions 4.3.0' to Application.Utils
Adding package 'System.Threading.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.4.3.0' to 'packages.config'
Successfully installed 'System.Threading 4.3.0' to Application.Utils
Adding package 'System.Threading.Tasks.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.Tasks.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.Tasks.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Tasks 4.3.0' to Application.Utils
Adding package 'System.Threading.Timer.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.Timer.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Threading.Timer.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Timer 4.3.0' to Application.Utils
Adding package 'System.Xml.ReaderWriter.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Xml.ReaderWriter.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Xml.ReaderWriter.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.ReaderWriter 4.3.0' to Application.Utils
Adding package 'System.Xml.XDocument.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Xml.XDocument.4.3.0' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'System.Xml.XDocument.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.XDocument 4.3.0' to Application.Utils
Adding package 'NETStandard.Library.1.6.1', which only has dependencies, to project 'Application.Utils'.
Adding package 'NETStandard.Library.1.6.1' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'NETStandard.Library.1.6.1' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'NETStandard.Library.1.6.1' to 'packages.config'
Successfully installed 'NETStandard.Library 1.6.1' to Application.Utils
Adding package 'JWT.3.1.1-beta' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'JWT.3.1.1-beta' to folder 'C:\Application\Source\Application.APP\packages'
Added package 'JWT.3.1.1-beta' to 'packages.config'
Successfully installed 'JWT 3.1.1-beta' to Application.Utils
Removing package 'JWT.3.0.1' from folder 'C:\Application\Source\Application.APP\packages'
Removed package 'JWT.3.0.1' from folder 'C:\Application\Source\Application.APP\packages'
Executing nuget actions took 11.17 sec
Time Elapsed: 00:00:34.4518312</sub>

**Comment by psylenced** [comment]
Thanks! Appreciate that quick turn around.

**Comment by abatishchev** [comment]
@nbarbettini  Hey Nate, is there a chance you could take a look and press the button?

**Comment by abatishchev** [comment]
Released to NuGet as [3.1.1](https://www.nuget.org/packages/JWT/3.1.1). Thanks you both!

**Pull request merged**